pkg/kubelet/prober/worker.go

```diff
@@ -105,13 +108,14 @@ func (w *worker) run() {
 		w.probeManager.removeWorker(w.pod.UID, w.container.Name, w.probeType)
 	}()
 
+	tickPeriod := time.Duration(w.spec.PeriodSeconds) * time.Second
 probeLoop:
 	for w.doProbe() {
 		// Wait for next probe tick.
 		select {
 		case <-w.stop:
 			break probeLoop
-		case <-probeTicker.C:
+		case <-time.Tick(wait.Jitter(tickPeriod, workerProbeJitterFactor)):
```

Comment:
`time.Tick` doesn't provide you any way to shut down the ticker, which means the underlying Ticker can never be GC'd. It's probably better to keep `probeTicker` if possible, isn't it?